rpp-charter.md

```diff
@@ -64,6 +64,7 @@ After finding consensus regarding requirements, the next documents to be deliver
 by this working group will describe the core architecture and JSON mapping of RPP with regard to the
 provisioning of domain names, followed by any specifications necessary
 to implement the core architecture and the JSON mapping for the different objects to be provisioned.
+These documents may then be followed by mappings between RPP and EPP.
```

Comment:
I think the "mappings between RPP and EPP" document must be moved.
new proposal:

"this working group will describe the core architecture and JSON mapping of RPP with regard to the
provisioning of domain names. These documents are then followed by a document describing the mappings between RPP and EPP. Followed by any specifications necessary to implement the core architecture and the JSON mapping for the different objects to be provisioned."

Rationale:
It may take some time (if ever) before other objects (numbers? widgets?) are described in documents, implementers need a mapping between RPP and EPP after the core architecture and JSON mapping of RPP  for domains names is completed.

Reply:
How does a mapping occur before any core specifications? It seems to me that if you want to map "blah": "foo" in JSON to <blah>foo</blah> in XML, you need concrete implementation specifications.

Reply:
no, i don't mean to say that the mapping from RPP to EPP comes first.

i think this is the correct order:

1: the core architecture and JSON mapping of RPP with regard to the provisioning of domain names
2: mappings between RPP and EPP.
3: any other (optional) object mapping, e.g. numbers.


because at the moment it is not clear that other object mapping (3) will ever happen and if it does how long this will take. Therefore, it's better to create the  mappings between RPP and EPP (2) immediately after the core architecture and JSON mapping of RPP (1)